lib/plugins/aws/lib/naming.js

```diff
@@ -58,6 +58,12 @@ module.exports = {
   getLambdaOutputLogicalIdRegex() {
     return /LambdaFunctionArn$/;
   },
+  getLambdaVersionLogicalId(functionName, sha) {
+    return `${functionName}Version${sha.replace(/[^0-9a-z]/gi, '')}`;
```

Comment:
Should I instead be using `normalizedFunctionName` rather than the plain function?